### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by Zykino** [aaronrancsik/prismedia#1]
Hi,

To be clear upfront, you can keep this mirror, LeCygneNoir and me are happy to found out someone else is using our software :heart:.

Now I’m contacting you out of curiosity more than anything else.
1) How do you mirror this repo? Manually? Automatically (How? I don’t know if tools exist)?
2) Why do you feel the need to mirror the repo on github? What is to you the advantage of Github over LeCygneNoir’s Gitea?

Best regards,
Zykino

**Comment by aaronrancsik** [comment]
Hello, 
First of all this stuff is absolutely amazing  :D especially In the age of covid and online learning.
1. Manually, for time to time I have to pull and push the modifications here.
2. I can't find a way to register here LeCygneNoir’s Gitea :facepalm: maybe it's my bad. 
My future plan is to keep this mirror update and create a fork of it here because Github is my main git platform.
So it's just a personal mirror for forking. My forking ideas are not complete yet and rn I haven't got much free time so this mirror maybe out of date already.. But some day it will just serve these purpose.

**Comment by Zykino** [comment]
Yeah LeCygneNoir had to inactivate inscriptions because there is too much bot :s. He is waiting on a Gitea update to have some captcha. Don’t worry we don’t update the soft too often, and the last hotfix introduced a bug :sweat_smile:. If you want to contribute please go ahead (if your idea is already in the issues listed we would need to communicate not to develop the same thing twice :smile:).
